client-course-schedulizer/src/utilities/services/addSectionService.ts

```diff
@@ -51,6 +52,13 @@ export interface SectionInput {
   year: string; // Assume string till yearCase() decides
 }
 
+export interface NonTeachingLoadInput {
+  facultyHours: Section["facultyHours"];
+  instructor: Instructor;
+  name: Course["name"];
```

Comment:
```suggestion
  activity: Course["activity"];
```

Reply:
Oops, this was a bad suggestion (good job realizing this should be `activity: Section["instructionalMethod"];`)